app-misc/peaclock/peaclock-0.1.3.recipe

```diff
@@ -0,0 +1,59 @@
+SUMMARY="A colourful binary clock for the terminal"
+DESCRIPTION="Features:
+* digital clock
+* binary clock
+* 12/24 hour time
+* customize with hex colour codes
+* set a custom unicode character for the binary clock graphic
+* compact or expanded mode
+* toggle visibility of digital and binary clocks
+* command prompt with a readline-like interface"
+HOMEPAGE="https://github.com/octobanana/peaclock"
+COPYRIGHT="2019 Brett Robinson"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="$HOMEPAGE/archive/$portVersion.tar.gz"
+CHECKSUM_SHA256="0041de0e50429786dff4152c60f6dbd84249287d289ad6d7573e47c9f9b62afc"
+SOURCE_FILENAME="peaclock-$portVersion.tar.gz"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+commandBinDir=$binDir
+if [[ $targetArchitecture == x86_gcc2 ]]; then
+SECONDARY_ARCHITECTURES="x86"
+commandBinDir=$prefix/bin
+fi
+
+PROVIDES="
+	peaclock$secondaryArchSuffix = $portVersion
+	cmd:peaclock
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:cmake
+	cmd:gcc$secondaryArchSuffix
+	cmd:make
+	"
+
+defineDebugInfoPackage peaclock$secondaryArchSuffix \
+	"$commandBinDir"/peaclock
+
+BUILD()
+{
+	mkdir -p build
+	cd build
+	cmake ../ -DCMAKE_BUILD_TYPE=RelWithDebInfo
+	make $jobArgs
+}
+
+INSTALL()
+{
+	install -d "$commandBinDir" "$docDir"
+	install -m 755 build/peaclock "$commandBinDir"
+	install -m 644 README.md "$docDir"
```

Comment:
This README seems to contain more information about how to build and install the software than using it, so I don't think we should include it.

Reply:
It does include a nice explanation of how to read it, though. But I guess that people wouldn't install it without knowing how to read it...